static/app/components/events/eventTags/index.tsx

```diff
@@ -80,10 +80,16 @@ export function EventTags({event, organization, projectSlug, location}: Props) {
   useEffect(() => {
     const mechanism = event.tags?.find(tag => tag.key === 'mechanism')?.value;
     const transaction = Sentry.getCurrentHub().getScope()?.getTransaction();
-    if (mechanism && transaction) {
-      transaction.tags.hasMechanism = mechanism;
+    if (mechanism) {
+      trackAnalytics('event_with_mechanism.viewed', {
+        organization,
+        mechanism,
+      });
```

Comment:
if this is going to emit on most issue details pages i would add this to `getAnalyticsDataForEvent` instead